### ISSUE TITLE
Required files for technology = smartseq2

### ISSUE THREAD

**Issue by ostkampp** [pachterlab/kallistobustools#38]
Hi,

I tried using kb-python to get raw count matrices from fastq files obtained from smartseq2. I got paired end data and have two fastq files. However, I got an error telling me I need four files. So what exactly does kb-python count expect when using technology smartseq2?

Best

**Comment by Yenaled** [comment]
Can you show us the exact command you're running and the error message?

**Comment by DarioS** [comment]
Would not a different toolkit be better-suited for Smart-seq2 data, since it sequences along the entire gene length?

**Comment by Yenaled** [comment]
No, kallisto is fine for that purpose (and was actually initially designed for full gene lengths)
